templates/main_page.html

```diff
@@ -27,6 +27,42 @@
             {% for type,color,count in data %}<div class="bar" style="background-color:{{color}};width:{{type}}%" title="{{count}}"></div>{% endfor %}
         </div>
 
+        <br>
+
+        <table class="table table-striped">
+            <thead>
+            <tr>
+                <th>Report</th>
+                <th>Plagiarized Post</th>
+                <th>Original Post</th>
+            </tr>
+            </thead>
+            <tbody>
+            {% for post_id, title_one, title_two, url_one, url_two in posts %}
+                <tr>
+                    <td>
+                        <a href="http://copypastor.sobotics.org/posts/{{post_id}}">{{post_id}}</a>
```

Comment:
The URL should not be hardcoded to copypastor.sobotics.org
I'd rather use `/posts/{{post_id}}`.

On the test-instance, this leads to an error 404 when clicking the report-ID: http://guttenberg.sobotics.org:5001